### PR TITLE
update profile

### PR DESCRIPTION
**Pull request by Collins-Omariba**
-  edited bio section 
- added tags

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2270"></a>

### CODE REVIEW

**Comment by Collins-Omariba** [comment]
The format check seems not to pass , what could be the problem?

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
> The format check seems not to pass , what could be the problem?

Problem was a space see https://github.com/EddieHubCommunity/LinkFree/pull/2270/commits/255006946bbdaeee16cb85fdfea08bf6c50a8705

File: data/Collins-Omariba.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   "type": "personal",
-
-  "bio": "Backend Web dev and Machine Learning Enthusiast!",
-
+  "bio": "Backend dev and Machine Learning",
   "avatar": "https://github.com/Collins-Omariba.png",
-
+  "tags": ["Open Source", "Python", "JavaScript", "ML", "IoT"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/OmaribaCollins " },
```
Comment (schmelto) [inline data/Collins-Omariba.json:8]:
```suggestion
    { "platform": "twitter", "url": "https://twitter.com/OmaribaCollins" },
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by Collins-Omariba** [comment]
Great 😃

**Pull request merged**